.expeditor/config.ym_

```diff
@@ -0,0 +1,55 @@
+# Documentation available at https://expeditor.chef.io/docs/getting-started/
```

Comment:
was removing the `.yml` file and having this `.ym_` file intentional?

Reply:
I had included it in case Releng created a way down the road to build a gem out automatically but just removed it to simplify